store/interfaces.json

```
@@ -9,12 +9,24 @@
     "component": "ImageSlider",
     "content": {
       "properties": {
-        "images": { "$ref": "app:vtex.store-image#/definitions/Images" }
+        "images": {
+          "$ref": "app:vtex.store-image#/definitions/Images"
+        }
       }
     }
   },
-  "list-context.image-list": {
+  "image-list": {
     "component": "ImageList",
+    "content": {
+      "properties": {
+        "images": {
+          "$ref": "app:vtex.store-image#/definitions/Images"
+        }
+      }
+    }
+  },
+  "list-context.image-list": {
+    "component": "ListContextImageList",
```

Changing the name of the files is considered a breaking change.

Example if someone is importing this file, with this PR the behaviour of this component changed:

```
import { ImageList } from 'vtex.store-image'
```

So my suggestion is to keep the old name here and give another name for the other interface (even though this name would be better, we should avoid breaking changes) 
```suggestion
    "component": "ImageList",
```

---

Missing changelog too

---

Updated the Changelog, updating the file naming right now. Good call, thank you!

---

I've updated the name of the files and I have also re-tested the functionality on this workspace:

[Link to demo of image-list without context](https://beta--fstudio.myvtex.com/lp1)
[Link to demo of image-list WITH context](https://beta--fstudio.myvtex.com/)